nilearn/surface/surface.py

```diff
@@ -424,6 +482,8 @@ def vol_to_surf(img, surf_mesh,
 
     kind : {'line', 'ball'}
         The strategy used to sample image intensities around each vertex.
+        If `inner_mesh` is provided this parameter is ignored and values are
+        sampled along a line
```

Comment:
Why not make a third kind 'depth' or similar? I find it not obvious otherwise that there is a third way to sample, which is invoked when inner_mesh is provided

Reply:
I added the 'depth' strategy, and made the default 'auto', which chooses 'depth' if 'inner_mesh' is provided and 'line' otherwise